### DISCUSSION TITLE
FoundationDB-related Question Discussion

### DISCUSSION THREAD

**Discussion by xiaomo728** [ByConity/ByConity#213] [Q&A] [answered]
**(20230410) Question 1**
As we just use FoundationDB's official docker images: https://hub.docker.com/r/foundationdb/build
This image is too big (after uncompressing about 17G).

How could I strip it?

**Comment by dmthuc** [comment, marked as answer]
this is a build image, not for running, you are using the wrong docker image

**Reply by liby888** [reply]
we just want to use the base foundation image to build . not for running

**Reply by dmthuc** [reply]
Hi @liby888 what do you want to build with foundationdb build image? You want to build foundationdb by yourself? The build image is supposed to be heavy because it contains many libraries and tool chain required to build foundationdb. It is heavy but you only need to pull the first time.

**Comment by hustnn** [comment]
https://hub.docker.com/layers/foundationdb/foundationdb/7.1.27/images/sha256-87931b6fd26ca71d4deec9f359f8e695e84119e9d19afb413e6b698a8dd198fc?context=explore

Looks like not so large, can you double check are you using the correct image?

**Reply by xiaomo728** [reply]
I will check for this. Thank you very much.
